NFT.md

```diff
@@ -48,13 +48,13 @@ As described above, a problem with the "one genesis per token" approach is that
 
 Create a fungible token using a GENESIS transaction; for optimal NFT quantity management, it is recommended that the fungible token to be created without decimal places. In addition to the usual fields, add "NFT1 " at the beginning of document URL to specify that this token can be used to mint NFT tokens. 
 
-To forge an NFT token, first prepare an UTXO with one or more of the NFT-enabled fungible tokens associated with it; then spend that UTXO in a GENESIS transaction that generates a new token with quantity of 1 and decimal places 0, prefixing its document URL with "NFT1_" followed by token hash of the fungible token for quick lookup. This simultaneously generates a new token and burn one fungible token. Wallets that support this scheme can check whether a given NFT token is valid by tracing DAG up to genesis, check if the tx burned any of the fungible token it specified, and check the validity of that back to the genesis of said fungible token. Each NFT token generated this way will have its own secondary "NFT id" from the txid of its genesis, and can remain distinct from each other.
+To forge an NFT token, first prepare an UTXO with one or more of the NFT-enabled fungible tokens associated with it; then spend that UTXO in a GENESIS transaction that generates a new token with quantity of 1 and decimal places 0, prefixing its document URL with "NFT1_" followed by token hash of the fungible token for quick lookup, followed by a two-byte number indicating which input contained the "generating" token in question. This simultaneously generates a new token and burn one fungible token. Wallets that support this scheme can check whether a given NFT token is valid by tracing DAG up to genesis, check if the tx burned any of the fungible token it specified, and check the validity of that back to the genesis of said fungible token. Each NFT token generated this way will have its own secondary "NFT id" from the txid of its genesis, and can remain distinct from each other.
```

Comment:
1. suggest changing "; then spend that UTXO in a GENESIS transaction" to. "; then burn/spend that token UTXO in a GENESIS transaction"

Reply:
2. why not just a 1-byte number indicating which input contained? Do you expect it will be greater than index 255?

Reply:
1. Good point! Should I make a follow up PR? You can also fix that at your convenience. 

2. Transactions can already have more than 255 inputs https://blockchair.com/bitcoin-cash/transaction/dd111ad72411041e2753382095dd41a580d33f489766100c5404d22c11807a1f ; while a two byte field is probably enough for all forseeable future, using the token ID is probably more robust.

Reply:
I think we can just leave it for now.  :) Thank you for this contribution, I think this idea is going to take NFTs to the next level!

In the Electron Cash we can add a method & UI to burn tokens.

After that's added then we can extend that burn method to allow the user to right-click on any token and select "Mint a NFT" from any fungible token.